### ISSUE TITLE
ROI selection is broken with 2D data sets

### ISSUE THREAD

**Issue by drdavella** [spacetelescope/cubeviz#318]
Steps to reproduce:

1. Create a moment map or collapsed cube
2. Display the new 2D data set in any viewer (will happen automatically after #317 is merged)
3. Create an ROI in the viewer displaying the 2D data set
4. Try to click and drag the ROI to a new position

Doing so will cause a new ROI and subset to be created in the same location as the original, instead of moving it to a new location.

**Comment by drdavella** [comment]
This is a duplicate of #354 and was fixed by #361.
